src/partial17.lib/partial17/Partial.h

```diff
@@ -170,6 +170,12 @@ struct Partial {
     }
     auto has(size_t i) const { return m_bits[i]; }
 
+    template<class T>
+    auto has(meta17::Type<T> = {}) const {

```

Comment:
Please do not add `meta17` namespace in code. Add a Using on top. This should make the code more natural and readable.